e2e/testdata/fn-render/validate-generated-resource/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: app-with-generator
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/set-namespace:unstable
```

Comment:
Is there a reason most test cases being added have `set-namespace` and `set-label` in their pipeline?

In general, each test case should test one particular thing, and not repeat anything covered by other test cases. Another way to think about it: test cases should be orthogonal to each other.

@droot

Reply:
I can take them out. They are here because I was following the trend of what I see in other test cases but they aren't necessary

Reply:
Yes, we could improve tests to be more targeted. Lets try to do it for new tests we are adding. @natasha41575 would suggest to remove the unnecessary functions.